TombEngine/Specific/configuration.cpp

```diff
@@ -269,22 +277,26 @@ bool SaveConfiguration()
 	}
 
 	if (g_Configuration.Bindings.empty())
-		g_Configuration.Bindings = BindingManager::DEFAULT_KEYBOARD_MOUSE_BINDING_PROFILE;
+		g_Configuration.Bindings = DEFAULT_KEYBOARD_MOUSE_BINDING_PROFILE;
 
 	// Set Input binding keys.
-	for (int i = 0; i < (int)InputActionID::Count; i++)
+	for (auto actionGroupID : SAVEABLE_ACTION_GROUP_IDS)
```

Comment:
If I understand correctly, now every action group is saved separately?

Reply:
This is supposed to ignore hardcoded keyboard/mouse bindings and do the same as before.

Just note I made an error somewhere and saving the config doesn't work ATM.

Reply:
Binding config loading's been fixed now.